Vagrantfile

```diff
@@ -16,13 +16,28 @@ Vagrant.configure(2) do |config|
   config.vm.provider "virtualbox" do |vb|
     # Customize the amount of memory on the VM:
     vb.memory = "2048"
+    vb.cpus = 4
   end
 
   config.vm.provision "shell", inline: <<-SHELL
      sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
      echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list
      sudo apt-get update
      sudo apt-get install -y linux-image-extra-$(uname -r)
-     sudo apt-get install -y docker-engine
+     sudo mkdir -p /etc/systemd/system/docker.service.d
+     # https://docs.docker.com/engine/admin/#troubleshoot-conflicts-between-the-daemonjson-and-startup-scripts
+ echo "
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd" > /etc/systemd/system/docker.service.d/docker.conf
+     sudo apt-get install -y docker.io
```

Comment:
Why installing `docker.io` this package is quite old and deprecated (Docker 1.13)

Reply:
It just worked out of the box, want me to update PR with docker-ce ?